milestone1/src/main/java/pt/ulisboa/tecnico/milestone1/pooler/Pooler.java

```diff
@@ -66,19 +62,19 @@ private void start_workers(int nmrWorkers) throws InstantiationException, Illega
             workers.add(worker);
         } catch(NoSuchMethodException noSuchMethodExceptionHandler) {
             System.out.println("Method `main_routine` does not exist in class " + workerClass.getName()
-                                + ".\nDoes this class extends class worker?"); // Provide some advice
+                    + ".\nDoes this class extends class worker?"); // Provide some advice
             return; // and stop executing
         }
 
-        start_workers(nmrWorkers+1);
+        startWorkers(nmrWorkers+1);
     }
 
     private void gracefullyStopThread(Thread worker) {
-	try {
+        try {
```

Comment:
Why is this indentation being changed?

Reply:
To be coherent with the rest of the code.